controller/lib/core/sensors.cpp

```diff
@@ -31,14 +31,21 @@ static DebugVar d2("dp_exhale", &dp_exhale, "Exhale diff pressure, cm/h2O",
                    "%.4f");
 static DebugVar d3("pressure", &pressure, "Patient pressure, cm/h2O", "%.4f");
 
-static float flow_inhale, flow_exhale, flow_delta, volume;
+static float flow_inhale, flow_exhale, flow_corrected, flow_uncorrected, volume,
```

Comment:
Can you use DebugFloat's?

Reply:
Sure.  IMO it's only marginally better though, so I'd like to leave in the TODO complaining about it.

Some ideas I've been considering

 - `static DebugVar<Pressure> dbg_pressure("pressure", "Patient pressure");  // help text automatically adds, "cm h2o".`
 - Overloading `operator=` so you can do `VolumetricFlow inflow = dbg_inflow = PressureDeltaToFlow(inflow_delta);`.  Not wild about this, it seems kind of magic.
 - ...macros?  lol